src/lt/objs/app.cljs

```diff
@@ -209,10 +209,10 @@
                       (cmd/exec! :window.zoom-reset)))
 
 (behavior ::add-platform-class
-                  :triggers #{:init}
-                  :reaction (fn [this]
-                              (dom/add-class (dom/$ :body) (name platform/platform))
-                              ))
+          :triggers #{:init}
+          :reaction (fn [this]
+                      (dom/add-class (dom/$ :body) (name platform/platform))
+                      ))
```

Comment:
perhaps out of scope, but while you are at it :might be worth nuking a newline here -)